numpy/_core/tests/test_limited_api.py

```diff
@@ -67,6 +67,12 @@ def install_temp(tmpdir_factory):
         "and Py_REF_DEBUG"
     ),
 )
+@pytest.mark.xfail(
+    sysconfig.get_config_var("Py_GIL_DISABLED"),
```

Comment:
This is the second instance of this. Before it proliferates further, how about adding
```python
NOGIL_BUILD = bool(sysconfig.get_config_var('Py_GIL_DISABLED'))
```
to `numpy/testing/_private/utils.py` (next to `IS_WASM)` and importing the variable from there in the tests that need it?